<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>111</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Email Administrador:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Dispositivos externos:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Instalaciones:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Firewall:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Conexiones:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Antivirus:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Actualizaciones:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Clave API</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Generaci贸n Informe:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>281</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Herramienta de auto diagn贸stico</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>158</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="test"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Desactivado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Activado</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>Desactivado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Solo de origen desconocido</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Activado</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_3">
      <item>
       <property name="text">
        <string>Desactivado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Activado</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_4">
      <item>
       <property name="text">
        <string>Desactivado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Activado</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_5">
      <item>
       <property name="text">
        <string>Desactivado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Solo Instalaci贸n</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Activado</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_6">
      <item>
       <property name="text">
        <string>Desactivado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Activado</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QTimeEdit" name="timeEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QComboBox" name="comboBox_8">
      <item>
       <property name="text">
        <string>Ninguna</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mostrar aviso</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Apagar PC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Desconectar red</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_9">
      <item>
       <property name="text">
        <string>Ninguna</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mostrar aviso</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Apagar PC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Desconectar red</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_7">
      <item>
       <property name="text">
        <string>Ninguna</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mostrar aviso</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Apagar PC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Desconectar red</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_10">
      <item>
       <property name="text">
        <string>Ninguna</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mostrar aviso</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Apagar PC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Desconectar red</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_11">
      <item>
       <property name="text">
        <string>Ninguna</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mostrar aviso</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Apagar PC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Desconectar red</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_12">
      <item>
       <property name="text">
        <string>Ninguna</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mostrar aviso</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Apagar PC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Desconectar red</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Acci贸n de respuesta</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
